src/project-manager.ts

```diff
@@ -483,13 +483,18 @@ export class ProjectManager implements Disposable {
 		}
 		if (!rootdirs.has('') && !this.configs.has('')) {
 			// collecting all the files in workspace by making fake configuration object
-			this.configs.set('', new ProjectConfiguration(this.localFs, '/', this.versions, '', {
+			const tsConfig: any = {
 				compilerOptions: {
 					module: ts.ModuleKind.CommonJS,
 					allowNonTsExtensions: false,
 					allowJs: true
 				}
-			}, this.traceModuleResolution, this.logger));
+			};
+			// if there is at least one config, giving no files to default one
```

Comment:
My idea here is:
- if there are no config files as all, let's make a default 'catch-them-all' config
- if there are some config files, let's make default one to handle situation when information is requested from a file that doesn't belong to any config (we should stay alive) but make it empty to exclude files which werent't expected to be compilable